### DISCUSSION TITLE
Can't compile the server with modules.

### DISCUSSION THREAD

**Discussion by deadcorpseeater** [azerothcore/azerothcore-wotlk#16391] [Guides & Tips]
Everytime the same problem:

23>game.vcxproj -> D:\Build\src\server\game\RelWithDebInfo\game.lib
25>------ Сборка начата: проект: worldserver, Конфигурация: RelWithDebInfo x64 ------
25>Building Custom Rule D:/AzerothCore-wotlk-with-NPCBots/src/server/apps/CMakeLists.txt
25>cmake_pch.cxx
25>Main.cpp
25>ACSoap.cpp
25>CliRunnable.cpp
25>RASession.cpp
25>WheatyExceptionReport.cpp
25>ServiceWin32.cpp
25>LINK : fatal error LNK1181: не удается открыть входной файл "..\..\..\modules\RelWithDebInfo\modules.lib"
25>Сборка проекта "worldserver.vcxproj" завершена с ошибкой.
26>------ Сборка начата: проект: ALL_BUILD, Конфигурация: RelWithDebInfo x64 ------
26>Building Custom Rule D:/AzerothCore-wotlk-with-NPCBots/CMakeLists.txt
========== Сборка: успешно выполнено — 24 , со сбоем — 2, в актуальном состоянии — 0, пропущено — 0 ==========
========== Сборка запущено в 11:10 PM, заняло 05:13,603 мин ==========

**Comment by vnasing** [comment]
和我的错误一样，使用任何模块都不能成功编译，，不用模块可以正常编译。你找到问题了，也给我说一下解决方法
